### DISCUSSION TITLE
Any plans to support Octane v2.x?

### DISCUSSION THREAD

**Discussion by gazzoy** [brefphp/laravel-bridge#125] [General]
I'm a quite new at github, so please let me know if this is not the place I should ask.

Was just wondering, any plans to support Laravel Octane v2.x? As of I understood, at this point bref/laravel-bridge only supports Octane v1.x while the latest Octane version is 2.x. That would be great tif bref/laravel-bridge supports Octane v2.x.

**Comment by tillkruss** [comment]
Yes it will be supported at some point.

v2.0 is just a server update, no significant changes.
